css/style.css

```diff
@@ -0,0 +1,462 @@
+
+body {
+  background-color: white;
+  color: #333;
+}
+
+header {
+  height: 43px;
+  background-color: blue;
+}
+
+aside {
+  margin: 300px 0;
+}
+
+footer {
+  border-top: 1px solid #000;
+  position: fixed;
+  width: 100%;
+  height: 50px;
+  line-height: 17px;
+  bottom: 0;
+  font-size: 15px;
+  text-align: center;
+  color: #000;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  background-color: #fff;
+}
+
+i {
+  vertical-align: middle;
+}
+
+.footer-note {
+  position: relative;
+  top: 15px;
+}
+
+.footer-more {
+  display: flex;
```

Comment:
Great work with your CSS! But let's keep in mind that for this project we're required to use **floats** and not the flex property! :+1: 

 
![image](https://user-images.githubusercontent.com/5157985/70671183-e487e600-1c59-11ea-8099-f87a4b526034.png)

- [ ] Let's replace all of the flex property instances with float and also it's sub-properties in your style.css file :wink: